src/main/java/Task/DeadLine.java

```diff
@@ -0,0 +1,46 @@
+package task;
+import java.time.LocalDateTime;
+
+/**
+ * The `DeadLine` class represents a deadline task in the BloopBot application.
+ * It extends the base `Task` class and inherits its properties and methods.
+ * A `DeadLine` task includes a description and a deadline represented by a "by" string.
+ * It can be marked as completed or not completed.
+ * This class provides methods to create and format `DeadLine` tasks.
+ *
+ * @author raydenlim
+ * @version 0.0.0
+ */
+public class DeadLine extends Task {
+    protected LocalDateTime byDateTime;
+    protected String byDescription;
```

Comment:
Is byDescription an appropriate name? Perhaps description is sufficient as there is no by string used to represent the description

Reply:
woops, ye, previously, I had a byDate. So that distinguishes things, but I've since deleted it. Thanks for the note!